pending/service.go

```diff
@@ -31,6 +39,32 @@ func DefaultService(docSrv documents.Service, repo Repository) Service {
 	}
 }
 
+// Create creates either a new document or next version of an anchored document and stores the document.
+// errors out if there an pending document created already
+func (s service) Create(ctx context.Context, payload documents.UpdatePayload) (documents.Model, error) {
+	accID, err := contextutil.AccountDID(ctx)
+	if err != nil {
+		return nil, contextutil.ErrDIDMissingFromContext
+	}
+
+	if len(payload.DocumentID) > 0 {
+		_, err := s.pendingRepo.Get(accID[:], payload.DocumentID)
+		if err == nil {
```

Comment:
Should we start being a more explicit on doc not found vs other DB errors? WDYT

Reply:
yes, but in this scenario, we will fail if the doc is found in the DB since we cant have multiple